### PR TITLE
Update Polish translation file

### PR DESCRIPTION
**Pull request by yokonrad**
Hi, small fixes in Polish translation file

### CODE REVIEW

**Comment by exelban** [comment]
Hi. Thanks.

File: Stats/Supporting Files/pl.lproj/Localizable.strings [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //  Localizable.strings
 //  Stats
 //
-//  Created by Serhiy Mytrovtsiy on 27/08/2020.
+//  Created by Yoko on 15/03/2021.
```
Comment (exelban) [inline Stats/Supporting Files/pl.lproj/Localizable.strings:5]: Why did you change this?

Comment (yokonrad) [reply Stats/Supporting Files/pl.lproj/Localizable.strings:5]: Because i saw in other language files that the authors of the modified versions have changed this line. You can skip this line if you don't want to change it.

Comment (exelban) [reply Stats/Supporting Files/pl.lproj/Localizable.strings:5]: I'm ok with update credentials if you are an author of translation or contribute a lot to the existing one.

Comment (yokonrad) [reply Stats/Supporting Files/pl.lproj/Localizable.strings:5]: Well, im using your app and it wont change for a long time :) I used menumeters for 6-7 years before. So i will try to update it every time when it will be needed to.

Comment (exelban) [reply Stats/Supporting Files/pl.lproj/Localizable.strings:5]: Thanks. I'm trying to keep Polish translation up to date (it's my primary language on the main machine))

Comment (yokonrad) [reply Stats/Supporting Files/pl.lproj/Localizable.strings:5]: Ah, okey. Didnt know :) Then just skip those lines.
